hangupsbot/plugins/_showme/__init__.py

```diff
@@ -0,0 +1,79 @@
+"""
+"showme" function to retrieve snapshots from security cameras or other URL's accessible to the hangupsbot server and
+sent them to the user.
+
+Config must specify aliases and urls which should include any nessisary auth.
+"""
+__LICENSE__ = """
+The BSD License
+Copyright (c) 2015, Daniel Casner
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
+
+* Redistributions in binary form must reproduce the above copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+"""
+__author__ = "Daniel Casner <www.artificelab.com>"
+
+import time
+import aiohttp, asyncio, io, logging
+import plugins
+
+logger = logging.getLogger(__name__)
+
+def _initalize(bot):
+    if bot.get_config_option("showme") is not None:
+        plugins.register_user_command(["showme"])
```

Comment:
The current standard is to register the command.
If the configuration is not there, either work without it, or fail when the command is executed.
Don't generate an error message just because the plugin was activated, as all plugins may be activated.